### ISSUE TITLE
Wrong start menu entry when installing Spyder 5 in Windows

### ISSUE THREAD

**Issue by andprosoft** [spyder-ide/spyder#16250]
I have installed Spyder 5.1.1 in a new environment with the command

`conda install spyder` .

In the start menu, it created a directory with the name "${DISTRIBUTION_NAME}", where Spyder can be found:

![spyder_start_menu](https://user-images.githubusercontent.com/79669161/130126702-c5957b51-b822-4888-a46c-641669a26ade.png)

Previous versions of Spyder were correctly linked in the "Anaconda 3" start menu directory.

**Comment by ccordoba12** [comment]
Hey @andprosoft, thanks for reporting. That's not something we can't solve here because those menu entries are created by Anaconda or Conda-forge.

How did you install Spyder?

**Comment by andprosoft** [comment]
Hi @ccordoba12, thanks for reaching out.

I have installed Spyder by first creating a new environment in the Anaconda Navigator GUI. Then i have installed Spyder in the anaconda prompt with `conda install spyder`.

I was thinking that this might be a Spyder issue, because I have installed Spyder 3 and Spyder 4 in the same way and both times they were linked in the correct start menu directory (Anaconda 3). So something must have happened between Spyder 4 and Spyder 5.
